packages/loot-core/src/server/cloud-storage.ts

```diff
@@ -348,11 +348,12 @@ export async function listRemoteFiles() {
       },
     });
   } catch (e) {
-    console.log('Error', e);
+    console.log('Error fetching file list from server', e);
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
    console.log('Unexpected error fetching file list from server', e);
```

Small suggestion: change the two `console.log` messages so in case a user encounters them - its easier to distinguish which one it is.